test/integration.lean

```diff
@@ -46,8 +46,7 @@ example : ∫ x in 0..2, -exp (-x) = exp (-2) - 1 := by norm_num
 example : ∫ x in 1..2, exp (5*x - 5) = 1/5 * (exp 5 - 1) := by norm_num
 example : ∫ x in 0..π, cos (x/2) = 2 := by norm_num
 example : ∫ x in 0..π/4, sin (2*x) = 1/2 := by norm_num [mul_div_comm, mul_one_div]
-example {ω φ : ℝ} (h : ω ≠ 0) : ∫ θ in 0..2*π, sin (ω*θ + φ) = ω⁻¹ * (cos φ - cos (2*π*ω + φ)) :=
-  by simp [h, mul_comm]
+example (ω φ : ℝ) : ω • ∫ θ in 0..π, sin (ω*θ + φ) = cos φ - cos (ω*π + φ) := by simp
```

Comment:
Why `•` and not `*`?
```suggestion
example (ω φ : ℝ) : ω * ∫ θ in 0..π, sin (ω*θ + φ) = cos φ - cos (ω*π + φ) := by simp
```

Reply:
That would require me to add `rw ← smul_eq_mul` before the `simp`, and I wanted to keep the proof as simple as possible.

I could potentially ameliorate this by adding special versions of those `comp` lemmas with `f : ℝ → ℝ` instead of `f : ℝ → E`, which should allow me to use `*` there and not `•`. Is that worthwhile?

Reply:
At first I thought not but perhaps it is in terms of enabling computation of common integrals by `simp`/`norm_num`.

Reply:
I think it's worth either doing that, or changing the proof to use `smul_eq_mul`. If these are examples, they should be written to help someone wanting to compute integrals, not to demonstrate the boundaries of what-we-can-currently-do-with-simp. (Which is, admittedly, impressive. :-)

Reply:
Okay, that's fair. I'll add the `rw ← smul_eq_mul` here so that we can go ahead with merging this, then I can open a new PR with the new lemmas and make any subsequent changes to this example in that pull request. 

My aim is that integrals like these should be computable by `simp` and it seems silly that such a simple part has to be pulled out as its own step, so I think adding versions of these lemmas specified to `ℝ → ℝ` is a good solution.

Reply:
I've made the change here and opened #7216.